documentation/builders/README.md

```diff
@@ -25,6 +25,7 @@
 
 * [Bluetooth (and audio buttons)](./bluetooth-audio-buttons.md)
 * [Auto Hotspot](./autohotspot.md)
+* [Samba](./samba.md)
```

Comment:
Some people might not even know what Samba is. Should we make it more explicit? We aslo call it "Audio" and not "pulseaudio" or "alsa"

First suggestion would be

> "Music file upload / Samba"

or 

> "File share / Samba"

Later, we could add the option to manage files via Web App if this has been built :-)

Reply:
See https://github.com/MiczFlor/RPi-Jukebox-RFID/pull/2216#discussion_r1464469003 for a suggestion